### PR TITLE
JH on opf

### LINKED ISSUE #75: Short Access JH DOC
Short "how to access JH on OPF" doc, adapted from   https://www.youtube.com/watch?v=v4mur-_ywBM&t=867s

### PR DESCRIPTION
**Pull request by aakankshaduggal**
Fixes #75

### CODE REVIEW

File: docs/JH_access.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Deploy Jupyter Lab or Notebooks easily with Operate First
+
+In this guide, we will go over how you can spin up a classic Jupyter Notebook or a Jupyterlab instance to get started with your very own cloud based data science development environment.
+
+
+* To get started, first go to the [operate-first](https://www.operate-first.cloud/) website.
```
Comment (MichaelClifford) [inline docs/JH_access.md:6]:
```suggestion
* To get started, first go to the [operate-first](https://www.operate-first.cloud/) website and navigate to `Community Handbook` > `GitOps Docs` > `Documentation`. Here you can find info about all the clusters and services managed by the OPF team. 
```

File: docs/JH_access.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+* To find the resources to access Jupyterhub and Jupyterlab instances, go  to [Open Data Hub](https://www.operate-first.cloud/apps/content/odh/README.html).
```
Comment (MichaelClifford) [inline docs/JH_access.md:8]:
```suggestion
* To find the resources to access JupyterHub, select `Open Data Hub (ODH)` from the menu on the left  to [Open Data Hub](https://www.operate-first.cloud/apps/content/odh/README.html). Then click on the [UI link]( https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud). 
```

File: docs/JH_access.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+* Here, you can find all the applications that are enabled on the cluster. Let’s start by clicking on the link for the [JupyterHub application](https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/).
```
Comment (MichaelClifford) [inline docs/JH_access.md:10]:
```suggestion

```

File: docs/JH_access.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+    ![Event logs](./../public/assets/Setup_Environment/event_logs.png)
+
+* When you launch your jupyterlab environment for the first time it will be empty. However, the instance here has connected to the storage volume associated with an existing account and so shows some projects that the user has been working on. This page provides a nice user-friendly UI to navigate the notebooks or other files you have in the remote storage. 
+
+    ![JH start page](./../public/assets/Setup_Environment/jh_start_page.png)
+
+* Now assuming your directory is completely empty, let’s go ahead and create a new notebook. We can click “+” in the upper left-hand corner, then select python3 from the notebook section. This will spawn a new empty “untitled” notebook for us.
```
Comment (MichaelClifford) [inline docs/JH_access.md:38]: Let's update this instruction for jupyter lab.

File: docs/JH_access.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+* You can rename and save the notebook and then shut it down. Once the notebook is closed, you can see that it’s still running on the server based on this green icon next to the file name. 
+
+    ![Running green icon](./../public/assets/Setup_Environment/green_icon.png)
+
+    ![Running Tab](./../public/assets/Setup_Environment/running_tab.png)
+
+* Going to the running tab, we can shut it down completely. 
+
+* This was the classic notebook environment that you’re probably used to.  We can also use a jupyter lab environment with its enhanced UI and a larger selection of extensions. Jupyter lab is great, it retains all the benefits of the classic notebooks but with a little more flexible and feature-rich UI as well as a whole host of additional extensions to make life easier for data science development.
```
Comment (MichaelClifford) [inline docs/JH_access.md:50]:
```suggestion
*You can also use a classic jupyter notebook environment, however, we  typically prefer the jupyter lab environment with its enhanced UI and a larger selection of extensions. Jupyter lab retains all the benefits of the classic notebooks but with a little more flexible and feature-rich UI as well as a whole host of additional extensions to make life easier for data science development.
```

**Review by MichaelClifford — CHANGES_REQUESTED** [review]

**Review by oindrillac — CHANGES_REQUESTED** [review]
Can we add a section on how to monitor your jupyter notebook workloads using Grafana dashboards here?
( see this section https://github.com/aicoe-aiops/data-science-workflows/pull/84#discussion_r765828776 which was removed from getting started)

File: docs/JH_access.md (hunk 4 above)
Comment (aakankshaduggal) [reply docs/JH_access.md]: Added :+1:

**Comment by aakankshaduggal** [comment]
> Can we add a section on how to monitor your jupyter notebook workloads using Grafana dashboards here? ( see this section [#84 (comment)](https://github.com/aicoe-aiops/data-science-workflows/pull/84#discussion_r765828776) which was removed from getting started)

Sure, I will add it to this document :+1:

**Review by oindrillac — COMMENTED** [review]
/lgtm

**Review by MichaelClifford — APPROVED** [review]

**Pull request merged**